test/libyul/yulOptimizerTests/fullSuite/abi_example1.yul

```diff
@@ -568,4 +565,83 @@
 //         if gt(length, 0xffffffffffffffff) { revert(0, 0) }
 //         size := mul(length, 0x20)
 //     }
+//     function outlined$1536$(array)
+//     { revert(array, array) }
+//     function outlined$2097$_284(dst, src, _1) -> dst_1, src_1
+//     {
+//         dst_1 := dst
+//         src_1 := add(src, _1)
+//     }
+//     function outlined$2378$_290(_1, length, src, dst, _2) -> src_1, dst_1
```

Comment:
Most of these are pretty silly due to the lack of an outlining heuristic so far - and it's probably not the best place in the suite, but it seems we do get larger matches this way.